module/cmd-plugins/src/main/java/net/md_5/bungee/module/cmd/plugins/PluginPlugins.java

```diff
@@ -0,0 +1,13 @@
+package net.md_5.bungee.module.cmd.plugins;
+
+import net.md_5.bungee.api.plugin.Plugin;
+
+public class PluginPlugins extends Plugin
```

Comment:
What should "PluginPlugins" mean? This class name is not very descriptive.

Reply:
I was trying to be consistent with other modules' main class naming
see https://github.com/SpigotMC/BungeeCord/blob/master/module/cmd-list/src/main/java/net/md_5/bungee/module/cmd/list/PluginList.java